composer.json

```diff
@@ -7,7 +7,8 @@
         "symfony/http-foundation": "2.8.2 as dev-master",
         "symfony/yaml": "2.8.2 as dev-master",
         "zendframework/zend-diactoros": "1.3",
-        "symfony/psr-http-message-bridge": "0.2.0"
+        "symfony/psr-http-message-bridge": "0.2.0",
+        "slim/slim": "3.0"
```

Comment:
Slim 3.0.0 requires PHP 5.5. We might be stuck with 2.6.2 as it is the last version compatible with <= 5.4